Doc/library/unittest.rst

```diff
@@ -2279,6 +2302,12 @@ Loading and running tests
    Calling ``main`` actually returns an instance of the ``TestProgram`` class.
    This stores the result of the tests run as the ``result`` attribute.
 
+   When *debug* is ``True`` (corresponding to ``--debug``) execution terminates
```

Comment:
You can link with `:option:--debug` I think

Reply:
Done